docs/pages/access-controls/sso/github-sso.mdx

```diff
@@ -40,6 +41,8 @@ Instructions for creating a GitHub OAuth app are available here:
 Define a GitHub authentication connector by creating a file called `github.yaml`
 with the following content:
 
+<ScopedBlock scope="oss">
```

Comment:
We prefer to use Tabs in cases like this for readers who forgot to (or don't realize they can) select their environment at the top of the page.

I'll submit a commit to adjust.

Reply:
Also, should the Enterprise version also apply to Cloud?